upload-s3/action.yml

```diff
@@ -0,0 +1,24 @@
+name: 'Upload S3'
+description: 'Uploads package to S3'
+inputs:
+  bucket:
+    required: true
+  product:
+    required: true
+  branch:
+    required: true
+  build:
+    required: true
+
+runs:
+  using: "composite"

```

Comment:
TIL - neat! I assume this run after `entrypoint.sh`?

Reply:
As I understand it, this doesn't run in a separate container - it runs in the same container as the workflow. It's basically a nice way of packaging steps together and making them reusable.

Reply:
Minor: would it be worth putting everything here or in `entrypoint.sh` so it's in one place?

Reply:
For now, I think it's simpler this way. If we add more stuff (for example, uploading the entire folder instead of a tarball), it might be cleaner to put everything in a single script.